src/test/scala/cromwell/OptionalParamWorkflowSpec.scala

```diff
@@ -35,4 +37,35 @@ class OptionalParamWorkflowSpec extends CromwellTestkitSpec("OptionalParamWorkfl
       )
     }
   }
+
+  "A workflow with an optional parameter that has a prefix inside the tag" should {
+    "not include that prefix if no value is specified" in {
+      val wf = """
+         |task find {
+         |  command {
+         |    find ${File root} ${"-name " pattern?}
+         |  }
+         |}
+         |
+         |workflow wf {
+         |  call find
+         |}
+       """.stripMargin
+      val ns = WdlNamespace.load(wf, BackendType.LOCAL)
+      val findTask = ns.findTask("find").getOrElse {
```

Comment:
Quickly scanning, I see three dot-infix `.xyz{` like this one, that should be converted to either `.xyz(` or ` xzy {`.